### ISSUE TITLE
多卡情况下paddle.nn.BatchNorm 是不是要替换成 paddle.nn.SyncBatchNorm

### ISSUE THREAD

**Issue by popova-2** [PaddlePaddle/Paddle#47557]
### 请提出你的问题 Please ask your question

多卡情况下paddle.nn.BatchNorm 是不是要替换成 paddle.nn.SyncBatchNorm

**Comment by ForFishes** [comment]
您好，这个和使用的算法有关。``paddle.nn.SyncBatchNorm``使用的统计量是来自所有卡的，而``paddle.nn.BatchNorm``使用的是各自卡。如果多卡训练下，想要对齐单卡，那么就需要``paddle.nn.SyncBatchNorm``。一般情况下，使用syncbn的精度要好一些，但是也有例外情况，这个和模型等相关。

**Comment by yeyupiaoling** [comment]
@ForFishes 有没有方便的做法，当使用单卡训练的时候使用`paddle.nn.BatchNorm`，多卡的时候自动使用`paddle.nn.SyncBatchNorm`
